.github/config.yml

```diff
@@ -0,0 +1,47 @@
+---
+#-------------------------------------------------------------------------------
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening this issue. A contributor will be by to give feedback soon. In the meantime, please review the [Layer5 Contributors' Welcome Guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit?usp=sharing), engage in the [discussion forum](https://discuss.layer5.io), and be sure to join the [community Slack](http://slack.layer5.io/).
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Yay, your first pull request! :thumbsup: A contributor will be by to give feedback soon. In the meantime, you can find updates in the [#github-notifications](https://layer5io.slack.com/archives/CLVT4TSG4) channel in the [community Slack](https://slack.layer5.io).
+
+  Be sure to double-check that you have signed your commits. Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
+
+#-------------------------------------------------------------------------------
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Thanks for your contribution to Meshery! :tada:
+   
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-cpx/master/.github/assets/images/welcome/meshery-celebration-dark.svg">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-cpx/master/.github/assets/images/welcome/meshery-celebration-light.svg">
```

Comment:
@Shyam-12, I overlooked it at first but,  interchange the logos for color mode, we need the white text  logo for dark mode and grey text logo for light mode

Reply:
Okay! Yeah, let me quickly fix it!